README.md

```diff
@@ -1,3 +1,79 @@
 # Kinobi
 
-TODO
+[![npm][npm-image]][npm-url]
+[![npm-downloads][npm-downloads-image]][npm-url]
+[![ci][ci-image]][ci-url]
+
+[npm-downloads-image]: https://img.shields.io/npm/dm/kinobi.svg?style=flat
+[npm-image]: https://img.shields.io/npm/v/kinobi.svg?style=flat
+[npm-url]: https://www.npmjs.com/package/kinobi
+[ci-image]: https://img.shields.io/github/actions/workflow/status/kinobi-so/kinobi/main.yml?logo=GitHub
+[ci-url]: https://github.com/kinobi-so/kinobi/actions/workflows/main.yml
+
+Kinobi is a tool that describes any Solana program in a powerful standardised format known as the Kinobi IDL. This IDL can then be used to create a variety of utility such as rendering client code for your programs in various languages/frameworks, generating CLIs, providing more information to explorers, etc.
```

Comment:
supper nit-pick: I think we don't need the ", etc." at the end of the sentence since we have "such as" just before providing the examples.